Formula/inlets.rb

```diff
@@ -0,0 +1,33 @@
+class Inlets < Formula
+  desc "Expose your local endpoints to the Internet"
+  homepage "https://github.com/alexellis/inlets"
+  url "https://github.com/alexellis/inlets.git",
+      :tag      => "0.5.7",
+      :revision => "4c5a519b669cfb92d3ced715fc8e82489aded46a"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["XC_OS"] = "darwin"
```

Comment:
is this required?

Reply:
You would need to tell me, as I said I copied the existing accepted and stable chart we produced for faas-cli. I will remove it given that you don't think it's needed.